cmake_modules/SetupCxxFlags.cmake

```diff
@@ -38,7 +44,7 @@ string(TOUPPER ${CMAKE_BUILD_TYPE} CMAKE_BUILD_TYPE)
 # Set compile flags based on the build type.
 message("Configured for ${CMAKE_BUILD_TYPE} build (set with cmake -DCMAKE_BUILD_TYPE={release,debug,fastdebug,relwithdebinfo})")
 if ("${CMAKE_BUILD_TYPE}" STREQUAL "DEBUG")
-    set(CXX_OPTIMIZATION_FLAGS "-ggdb -O0 -fno-omit-frame-pointer -fno-optimize-sibling-calls")
+    set(CXX_OPTIMIZATION_FLAGS "-ggdb -Og -fno-omit-frame-pointer -fno-optimize-sibling-calls")
```

Comment:
Debugging experience turned to crap when trying to step around in `void PlanGenerator::Visit(const InnerNLJoin *op)` calling out to `parser::ExpressionUtil::EvaluateExpression`. I vote we don't use this.

Reply:
Reverted.